Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/model/ContactDiaryPersonEncounter.kt

```diff
@@ -0,0 +1,12 @@
+package de.rki.coronawarnapp.contactdiary.model
+
+import org.joda.time.LocalDate
+
+interface ContactDiaryPersonEncounter {
+    val id: Long
+    val date: LocalDate
+    val contactDiaryPerson: ContactDiaryPerson
```

Comment:
Remark: wouldn't `person` be enough?

Reply:
yes would be enough, but to be aligned with ContactDiaryLocation I would like to let it this way. changing this would also require refactoring.